project/src/components/Pages/About/Sections/Journey/index.tsx

```
@@ -0,0 +1,64 @@
+            </div>
+        </section>
+    );
+}
+
+export function JourneyCard({ startDate, endDate, title, place, description }: JourneyCardProps) {
+    return (
+        <motion.div {...fadeInUpConfig} transition={{ ...fadeInUpConfig.transition, delay: 0.18 }} className='relative flex gap-2'>
+            {/* Timeline line and circle */}
+            <div className='relative flex flex-col items-center'>
+                <div className='h-4 w-4 rounded-full bg-cyan-400'></div>
+                <div className='h-full w-0.5 bg-gradient-to-b from-cyan-400/50 to-transparent'></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)